_posts/2020-06-12-status-update.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "12th June 2020: IAC Team Status Update"
+author: carabasdaniel
+categories:
+  - team
+  - status
+tags:
+  - litmus
+  - gpg
+  - puppet-modulebuilder
+  - community
+  - modules
+  - tools
+  - social
+  - archived
+---
+
+The IAC team has been working on a lot of things over the past week. We want to share with our community some of the most interesting things. Here’s a brief overview of what we’ve been up to:
+
+## Module Releases from IAC Team
+The release process for the following modules has been started and these will finished as soon as we manage to resolve our testing infrastructure issues:
+- [puppetlabs/postgresql](https://github.com/puppetlabs/puppetlabs-postgresql) (6.6.0)
+- [puppetlabs/ibm_installation_manager](https://github.com/puppetlabs/puppetlabs-ibm_installation_manager) (2.2.0)
+
+## Updated Puppet GPG signing key, 2020 edition
+
+The GPG signing key we’ve been using to sign Puppet products on Linux is expiring in August 2021 and this change will affect both Puppet Enterprise users and open source Puppet users. Please read the blog [here](http://feedproxy.google.com/~r/planetpuppet/~3/BAyZ9YYraQA/) for more information on how to update your keys.
+
+## Puppet Module Repository Housekeeping
+
+This week Puppet has started archiving stale unmaintained module repos living within the Puppet namespace.  
+The following module repos have been archived:
```

Comment:
```suggestion
Some examples of module repos that have been archived:
```

I don't think those are the only ones archived, but a good sampling of the crud going out.